docs/dashboard/orders.md

```diff
@@ -22,16 +22,39 @@ To process an order in the system, click _Finalize_ in the footer. Once the orde
 
 The main page shows a list of all orders in the system, regardless of their status.
 
-![](screenshots/order-list.jpg)
+![](screenshots/order-list.png)
 
 ### Order filters
 
-To automatically filter orders, click _Filters_ and view orders by date, customer or fulfillment status. Sorting by _status_ brings up a second menu from which you can choose _Cancelled_, _Fulfilled_, _Partially Fulfilled_, _Unfulfilled_, _Ready to capture_, _Ready to fulfill_.
+To automatically filter orders, click _Filters_ and view orders by channel, date, customer or fulfillment status. Sorting by _status_ brings up a second menu from which you can choose _Cancelled_, _Fulfilled_, _Partially Fulfilled_, _Unfulfilled_, _Ready to capture_, _Ready to fulfill_.
 
 You can also manually search specific information, such as key words or names, using the Search Orders field. The filter is dynamically applied as you type.
 
 ![](screenshots/order-filters.png)
 
+## Order settings
+
+:::note These options are available in Saleor 3.1 (alpha version). We recommend using the latest Saleor 3.0 beta version for production.
+:::
+
+You can manage your order and fulfillment settings in the order settings page. Click the three dot icon on top of the right corner of the main order list page. Then click _Order Settings_.
+
+### Settings
```

Comment:
There's also separate page for the order settings:
https://saleor-docs-git-new-fulfillment-settings-saleorcommerce.vercel.app/docs/3.0/dashboard/configuration/order-settings

Shouldn't that description land there?

Reply:
If content should be duplicated, then please update linked page

Reply:
@krzysztofwolski Good catch. IMO, This part you mentioned should be removed--as order settings are not a part of  `CONFIGURATION`